docker-app/qfieldcloud/filestorage/migrate_project_storage.py

```diff
@@ -165,6 +217,25 @@ def migrate_project_storage(
                     legacy_version_id=file_version.id,
                 )
 
+                # check that etags before and after are the same.
+                legacy_storage_etag = get_s3_object_etag(
```

Comment:
Is there a reason to prefer direct access to the S3 obejct storage instead of using the already available data in `file_version.e_tag`?